src/content/docs/journey-migration/making-the-switch.mdx

```diff
@@ -0,0 +1,127 @@
+        * Automatically instrument distributed tracing to start observing your data almost instantly.
+        * Utilize the seemless integration with our APM agents and let the two features work in tandem to reduce your workload.
+        * Capture representative activity samples with our head-based sampling approach.
+        * Take advantage of our **Infinite Tracing**, a feature that users tail-based sampling approaches that gathers information at the end of a trace and lets you know what's most important.
+    Learn more about distributed tracing in our [introduction to distributed tracing](/docs/distributed-tracing/concepts/introduction-distributed-tracing/) documentation.       
+    </Collapser>
+    <Collapser
+        id="NRQL"
+        title="NRQL: New Relic's query language"
+    >
+    Maximizing your data requires the ability to query what you need down to the smallest detail, and that's exactly what New Relic's query language (NRQL) provides. Similar to ANSI SQL, you can use our language as a foundation to get any number of details about your data in our platform.
+        * Build charts, dashboards, and alerts from anywhere in our platform using the NRQL console. 
+        * Take advantage of our free tutorials and New Relic University courses to learn everything you need to know about the language. 
+        * Explore your data in ways no other platform can provide and get every detail on any aspect of observability.
+        * Use NRQL to make queries and perform specific configurations via NerdGraph, our GraphQL-format API.
+    Learn more about [how NRQL works](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works/), or see our [introduction to NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) documentation.       
+
+    </Collapser>
+    <Collapser
+        id="Vuln-management"
+        title="Vulnerability management"
+    >
+    As your stack grows, security issues become an ever growing problem to tackle. Our vulnerability management solution gives you insight into what you need so that you can make sure to maintain compliance across your entire system.
+        * Sync up security with all of your teams from one centralized location and streamline issue management.
+        * Prevent accidental vulnerabilities and breaches across your entire organization with automatic identification. 
+        * Detect your vulnerabilities in minutes, letting you fix them before they grow into large-scale problems.
+        * See recommended updates automatically and ensure you're up-to-date on all your security needs.
+    Learn more about vulnerability management in our [short overview](https://newrelic.com/platform/vulnerability-management), or see our [introduction to vulnerability management](/docs/vulnerability-management/overview/) documentation.     
```

Comment:
Same as browser, gets combined with previosu bullet.

Reply:
Should be fixed